src/languages/golfscript/index.ts

```diff
@@ -89,8 +99,8 @@ const golfscriptLanguage: Language = {
         (node, spine) =>
           !isSubtype(getType(node.start, spine.root.node), integerType(0)),
       ),
-      implicitlyConvertPrintArg,
       replaceToSplitAndJoin,
+      useBuiltinAliases({ "\n": "n" }),
```

Comment:
This doesn't need to be applied as search, right?

Reply:
My understanding was it can be in required since it always makes the code strictly shorter.

Reply:
> My understanding was it can be in required since it always makes the code strictly shorter.

Yes, sorry misread it as being applied as search.

Reply:
But theoretically since it is not required to produce valid gs, it should be `simplegolf`.

Reply:
right, ill fix